src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMemberRow.jsx

```diff
@@ -106,70 +94,61 @@ export default class OrganizationMemberRow extends React.PureComponent {
     const detailsUrl = recreateRoute(id, {routes, params});
     const isInviteSuccessful = status === 'success';
     const isInviting = status === 'loading';
-    const canResend = !expired && canAddMembers && (pending || needsSso);
+    const showResendButton = pending || needsSso;
 
     return (
-      <PanelItem align="center" p={0} py={2}>
-        <Box pl={2}>
+      <StyledPanelItem>
+        <MemberHeading>
           <Avatar size={32} user={user ? user : {id: email, email}} />
-        </Box>
+          <MemberDescription onClick={() => browserHistory.push(detailsUrl)}>
```

Comment:
Let's make this a `Link`, otherwise it breaks usability (e.g. you can't view the link you're going to, or being able to right click and open in a new tab).